README.md

```diff
@@ -1,10 +1,18 @@
+<br/>
+
+> 👋 Welcome! The toolkit's [component library](./packages/component-library) is in preview; this means the API is subject to change without notice. Keep an eye on [Announcements](./discussions/categories/announcements) or [Releases](./releases) for news on releases, or check out our [Roadmap](./projects/5) for priorities.
+>
+> If you're interested in production use of this codebase, we'd love to chat! Please reach out at [toolkit@button.is](mailto:toolkit@button.is).
```

Comment:
let me make sure we have this shared inbox provisioned...

Reply:
I think we can merge this now - sent a test email to the new inbox, which worked :)